index.html

```diff
@@ -0,0 +1,157 @@
+          <div class="top-nav-bar">
+            <a href="#"><img class="top-nav-item" src="assets/facebook.png" alt="facebook"></a>
+            <a href="#"><img class="top-nav-item" src="assets/twitter .png" alt="twitter"></a>
+            <a href="#" class="top-nav-item">English</a>
+            <a href="#" class="top-nav-item">My Page</a>
+            <a href="#" class="top-nav-item">Logout</a>
+          </div>
+          <div class="bottom-nav-bar">
+            <a href="index.html"><img class="nav-bar-logo" src="assets/ailogo.png" alt="AI-logo"></a>
+            <div class="nav-bar-links">
+              <a class="bottom-nav-link" href="about.html">About</a>
+              <a class="bottom-nav-link" href="#partner">Partner</a>
+              <a class="bottom-nav-link" href="#">Join</a>
+              <a class="bottom-nav-link" href="#">Sponsor</a>
+              <a class="bottom-nav-link" id="ai" href="#">BUY TICKETS</a>
+            </div>
+          </div>
+          <!--  mobile-menu-->
+          <header class="mobile-head">
+            <nav class="mobile-nav">
+              <div class="mob-menu-links">
+                <img class="logo" src="assets/ailogo.png" alt="AI-logo">
+                <a href="#" class="close-link">x</a>
+                <a class="bottom-nav-link" href="about.html"><strong>About</strong></a>
+                <a class="bottom-nav-link" href="#main"><strong>Program</strong></a>
+                <a class="bottom-nav-link" href="#speakers"><strong>Sponsor</strong></a>
+                <a class="bottom-nav-link enroll" href="#"><strong>Buy Ticket</strong></a>
+              </div>
+              <img class="nav-bar-icon" src="assets/hamburger .png" alt="hamburger-icon">
+            </nav>
+          </header>
+          <!-- finish mobile menu -->
+      </nav>
+    </header>
+    <section class="info">
+      <div class="header-main">
+        <div class="header-main-content">
+          <p class="header-main-salutation">"Hello! World Sharing"</p>
+          <h1 class="header-main-title">
+            <strong>AI For GOOD </br>INDIA SUMMIT</br>2023 </strong>
+          </h1>
+          <p class="header-main-description">
+            The Artificial Intellegence for Good India Summit is the leading action-oriented United Nations platform
+            promoting AI to advance health, climate, gender, inclusive prosperity, sustainable infrastructure, and other
+            global development priorities.
+          </p>
+          <h2 class="header-date">
+            <strong>24 JULY 2023, 17:00-18:30(CEST GENEVA)</strong>
+          </h2>
+          <p class="header-main-subdescription">
+            Join <strong>India's </strong> vision to support united nations in the field of artificial intelligence
+          </p>
+        </div>
+      </div>
+    </section>
+  </div>
+  <!-- finish header section -->
+  <section class="program" id="program">
+    <h2 class="program-title"><strong>Main Program</strong></h2>
```

Comment:
- [OPTIONAL] As mentioned by the previous reviewer. Kindly add the red line under the titles to match the Figma design.
<img width="311" alt="251657352-75451004-95df-4b37-8b85-ed02a81c5320" src="https://github.com/the-faizmohammad/conferencePage/assets/91142432/6626287b-37e6-470f-8dd8-e52805bf8369">